scripts/spinup_comfystream_tensordock.py

```diff
@@ -96,9 +101,52 @@ def format(self, record):
 
 
 geolocator = Nominatim(user_agent="tensordock_locator", timeout=5)
+console = Console()
 
 
-def generate_strong_password(length: int = 65) -> str:
+def display_login_info(
+    comfyui_url: str,
+    comfyui_username: str,
+    comfyui_password: str,
+    ssh_command: str,
+):
+    """Display VM login information.
+
+    Args:
+        comfystream_ui_url: URL to the Comfystream UI.
+        comfystream_server_url: URL to the Comfystream Server.
+        ssh_command: SSH command to access the VM.
+        comfyui_url: URL to the ComfyUI (if exposed
+    """
+    comfyui_url_text = Text("ComfyUI url: ", style="yellow")
+    comfyui_url_text.append(f"{comfyui_url}", style="white")
+    comfyui_username_text = Text(f"ComfyUI username: ", style="cyan")
+    comfyui_username_text.append(comfyui_username, style="white")
+    comfyui_password_text = Text(f"ComfyUI password: ", style="cyan")
+    comfyui_password_text.append(comfyui_password, style="white")
+    ssh_text = Text("SSH Command: ", style="green")
+    ssh_text.append(ssh_command, style="white")
+
+    content = Text.assemble(
+        comfyui_url_text,
+        "\n",
+        comfyui_username_text,
+        "\n",
+        comfyui_password_text,
+        "\n",
+        ssh_text,
+    )
+    console.print(
+        Panel(
+            content,
+            title="[blue]Access Information[/blue]",
+            border_style="blue",
+            expand=False,
+        )
+    )
+
+
+def generate_strong_password(length: int = 60) -> str:
```

Comment:
Do we have to generate a password for them? Maybe we can create an interactive setup prompt in a future change